qiskit/validation/__init__.py

```diff
@@ -19,25 +19,19 @@
 
 .. currentmodule:: qiskit.validation
 
-Base
-====
+JSON Schema
+===========
 
-.. autosummary::
+,, autosummary::
```

Comment:
Did you mean to change this from dots to commas?

Reply:
oops, nope. I'll fix that in a sec

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/4030/commits/514d58fef10d2e3db29bfe4c7a71fb1ddf121c9a